.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration file was automatically generated by Gitpod.
```

Comment:
Moving files into a `.gitpod` directory causes Gitpod to just use it's default configuration file.

Let's simplify this whole thing, and start with the default Gitpod configuration file, and customize from there.

The tools that are currently installed in the default config are:

- `nvm --version`: `0.39.0`
- `node -v`: `16.13.2`
- `yarn --version`: `1.22.17`
- `git --version`: `2.35.1`
- `java --version`:
    ```
    Picked up JAVA_TOOL_OPTIONS:  -Xmx3435m
    openjdk 11.0.13 2021-10-19 LTS
    OpenJDK Runtime Environment Zulu11.52+13-CA (build 11.0.13+8-LTS)
    OpenJDK 64-Bit Server VM Zulu11.52+13-CA (build 11.0.13+8-LTS, mixed mode)
    ```
- `clang --version`:
  ```
  Ubuntu clang version 15.0.0-++20220226052900+e897214e97f2-1~exp1~20220226172958.171
  Target: x86_64-pc-linux-gnu
  Thread model: posix
  InstalledDir: /usr/bin
  ```
- `gcc --version`:
  ```
  gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
  ```
- `gcmake --version`: `cmake version 3.22.3`

Reply:
sounds good, no need to customize